content/applications/websites/ecommerce/checkout_payment_shipping/checkout.rst

```diff
@@ -2,133 +2,204 @@
+:ref:`Extra Info <ecommerce/checkout/extra_info>` step to the checkout process.
+
+You can also enable this feature by going to
+:menuselection:`Website --> Configuration --> Settings`, scrolling to the
+:guilabel:`Shop - Checkout Process` section, enabling :guilabel:`Extra Step During Checkout` and
+clicking :guilabel:`Save`. Click :icon:`fa-arrow-right` :guilabel:`Configure Form` to customize the
+form.
+
+Suggested accessories
+---------------------
+
+Toggle the :guilabel:`Suggested Accessories` feature to showcase
+:ref:`accessory products <ecommerce/cross_upselling/accessory>` at the
+:ref:`Review Order <ecommerce/checkout/review_order>` step.
 
-The :guilabel:`Extra Info` step is an online form linked to the quotation or sales order of the
-customer. The information added during that step can be found on the quotation or sales order of the
-customer from the back end, in the **Sales** app.
+.. _ecommerce/checkout/promo-code:
 
-When enabled, you can remove, add, and modify fields of the form by clicking on :guilabel:`Edit` in
-the top-right corner, and then clicking on any of the form's fields. All customization options, as
-well as the :guilabel:`+ Field` button to add new fields, are available at the bottom of the
-:guilabel:`Customize` menu under the :guilabel:`Field` section.
+Promo code
+----------
 
-.. image:: checkout/checkout-form.png
-   :align: center
-   :alt: Online form customization
+Enable the :guilabel:`Promo Code` option so that customers can redeem
+:ref:`gift cards <ewallet_gift/gift-cards>` or
+:doc:`discount <../../../sales/sales/products_prices/loyalty_discount>` codes.
 
-Confirm order: terms and conditions
------------------------------------
+Accept Terms & Conditions
+-------------------------
 
-You can ask customers to agree to the :guilabel:`Terms & Conditions` in order to confirm their order
-by enabling :guilabel:`Accept Terms & Conditions` under :menuselection:`Edit --> Customize` on any
-checkout page.
+Toggle :guilabel:`Accept Terms & Conditions` to ask customers to agree to the
+:doc:`terms and conditions <../../../finance/accounting/customer_invoices/terms_conditions>` before
+confirming their order.
 
 .. image:: checkout/checkout-terms.png
-   :align: center
-   :alt: Terms and conditions
+    :alt: Accept terms and conditions feature
+
+Show B2B Fields
+---------------
+
+Enable the :guilabel:`Show B2B Fields` option to add optional :guilabel:`VAT` and
+:guilabel:`Company name` fields to the :guilabel:`Billing address` form for B2B customers during the
+:ref:`Delivery <ecommerce/checkout/delivery>` step.
+
+Add to Wishlist
+---------------
+
+Activate :guilabel:`Add to Wishlist` to display the :guilabel:`Save for Later` option at the
+:guilabel:`Review Order` step to allow customers to remove a product from their order and save it in
+their Wishlist.
+
+.. image:: checkout/checkout-wishlist.png
+   :alt: Save for later option
+
+.. _ecommerce/checkout/express_checkout:
 
 Express checkout
```

Comment:
We should not use the term "express checkout" for the "buy now" feature. Express checkout refers to the express payment methods such as Apple Pay and Google Pay that can be displayed on the cart page, enabling the customer to skip entirely the Delivery and Payment checkout steps.